### PR TITLE
[api] API v2.3.0 Documentation

### PR DESCRIPTION
**Pull request by abellotti**
- Updated Versioning page to talk about v2.3.0 and its features
- /api/automate model queries
- /api/automate_domains queries and related actions
- OPTIONS on /api/:collection for fetching collection metadata
- Service Power operations
- Picture Creation via /api/pictures
- Deleting Service Dialogs
- Bulk Queries
- /api/notifications Support
- /api/blueprints Support and Tagging
- Provider custom attributes
- Entrypoint Details
- /api/arbitration_profiles CRUD
- /api/arbitration_rules CRUD
- /api/arbitration_settings CRUD
- /api/virtual_templates Queries
- Cloud Networks subcollection of Providers
- Service Creation
- /api/orchestration_templates CRUD
- Renamed/Removed Deprecated Attributes
- Approving/Denying service requests
- /api/requests Create/Edit/Approve/Deny

### CODE REVIEW

**Comment by abellotti** [comment]
@imtayadeway @mfeifer please review content/english when you get a chance.

/cc @Fryguy @gtanzillo

File: api/appendices/collection_metadata.adoc [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+| attributes | Physical attributes of the collection
+| virtual_attributes | Virtual attributes of the collection
+| relationships | Physical or Virtual associations to other resources
+| data | Additional meta-data for the collection
+|===================================================================
+
+The following is an example for the *groups* collection:
+
+----
+OPTIONS /api/:collection
```
Comment (imtayadeway) [inline api/appendices/collection_metadata.adoc]: Should this be `OPTIONS /api/groups`?

File: api/appendices/collection_metadata.adoc [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    "miq_user_role",
+    "miq_widget_contents",
+    "miq_widget_sets",
+    "taggings",
+    "tags",
+    "tenant",
+    "users",
+    "vms"
+  ],
+  "data": {
```
Comment (imtayadeway) [inline api/appendices/collection_metadata.adoc]: Do we have an example of what `data` is used for?

**Comment by abellotti** [comment]
Thanks @imtayadeway updated as per your comments and pushed updated PR.

**Comment by imtayadeway** [comment]
@abellotti content looks good. Great effort! I avoided looking at English as @mfeifer is better qualified to do that :+1:

**Comment by abellotti** [comment]
Thanks @imtayadeway 

@mfeifer when you get a chance for ye-old English.

File: api/appendices/collection_metadata.adoc (hunk 2 above)
Comment (mfeifer) [inline api/appendices/collection_metadata.adoc]: 👍

File: api/overview/bulk_query.adoc [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+[cols="1",options="header",width="40%"]
+|=======================================================================
+| Identifying Attributes
+| href
+| id
+| guid  (_if applicable_)
+|=======================================================================
+
+Select other attributes can also be used as identifying attributes
+as shown in the following table:
```
Comment (mfeifer) [inline api/overview/bulk_query.adoc]: Do you mean "Some attributes can also be used....."?

File: api/overview/filtering.adoc [hunk 4]
```diff
@@ -3,22 +3,22 @@
...
 ----
 
 Syntax for a filter is as follows:
 
 ----
-filter[]="attribute op value"
+filter[]=attribute op value
 ----
 
 By default, filters are logically anded together, for a logical or, prefix the attribute with "or" as follows:
```
Comment (mfeifer) [inline api/overview/filtering.adoc]: Might want to italicize, quote, or bold "anded" and "or".

**Comment by abellotti** [comment]
Thanks @mfeifer for the review, I've updated and re-pushed the PR.

**Comment by mfeifer** [comment]
LGTM

**Comment by abellotti** [comment]
@adahms Hi Andrew, we need this PR merged when you get a chance.  Thanks!!

/cc @mfeifer @imtayadeway

**Comment by abellotti** [comment]
ping @adahms

**Comment by adahms** [comment]
@abellotti 

Looks good to me; merging!

**Pull request merged**